### ISSUE TITLE
A11y_.NET Core_WPF_EditExaminer_ScreenReader: Narrator is not announcing the tooltip information for "Immediate Window", "Type command here", "No exception" control.

### ISSUE THREAD

**Issue by v-inanda** [dotnet/wpf#7874]
### Description

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS;#A11yTCS;#BM_.NET Core[WPF]_Win32_May23;#.NET Core;#A11ySev2;#WCAG1.3.1;#Narrator;#NVDA;#DesktopApp;#Win11;#Element:Tooltip;#Benchmark;

### Environment Details:
Application Name: .NET Core
.NET Version: .NET-SDK-8.0.100-preview.4.23260.5
WPF
Operating System: Windows 11 Enterprise 21H2
Screen Reader: Narrator, NVDA Version: 2022.4 (2022.4.0.27401)

### Reproduction Steps

1. Launch VS 2022 Preview
2. Navigate to Sample Applications and right click on the "Editing Examiner Demo" and click on build.
3. Then right click on Editing Examiner Demo then Click on Debug-->Start New Instance.
4. "Edit Examiner" screen should open.
5. Turn on Narrator.
6. TAB to "Immediate Window", "Type command here", "No exception" control.
7. Check whether Narrator is announcing the tooltip information for "Immediate Window", "Type command here", "No exception" control or not.

### Expected behavior

The tooltip information for "Immediate Window", "Type command here", "No exception" control should be read by screen reader.

### Actual behavior

Narrator is not announcing the tooltip information for "Immediate Window", "Type command here", "No exception" control.

Issue is also observed with NVDA screen reader.

![A11y_ NET Core_WPF_EditExaminer_ScreenReader](https://github.com/dotnet/wpf/assets/91525824/e62dba93-d95c-4143-a5d0-bcfae1ee4af4)

![A11y_ NET Core_WPF_EditExaminer_NVDA](https://github.com/dotnet/wpf/assets/91525824/bb502018-75b5-4211-b3a7-0d7985e72c4d)

### Impact

Screen reader users may not know the extra information which is provided in tooltip, and they will face difficulty in knowing the extra information about the control as screen reader is not announcing the tooltip information.

**Comment by harshit7962** [comment]
As we have discussed, we cannot reproduce the issue. I hope we can close this.

**Comment by v-inanda** [comment]
@harshit7962 Issue is fixed. 
Hence closing the bug.
![Presentation1](https://github.com/dotnet/wpf/assets/91525824/55fa02ca-6f7e-478b-9008-06f3ff5d28ce)
